apps/full-site-editing/README.md

```diff
@@ -4,37 +4,32 @@ This app contains both the `full-site-editing-plugin` and the required `blank-th
 
 ## File Architecture
 
-These is an example of what we should find inside the theme and plugin folders:
-
 ```
 /blank-theme
-  /dist
-    blank-theme.css
-    blank-theme.js
   functions.php
   index.js
   index.php
   index.scss
 
 /full-site-editing-plugin
-  /dist
-    full-site-editing-plugin.css
-    full-site-editing-plugin.js
   full-site-editing-plugin.php
   index.js
   index.scss
-```
-
-## Local Development
 
-Symlink both the theme and the plugin into a local WordPress install.
-
-E.g.
-
-```
-ln -s ~/Dev/wp-calypso/apps/full-site-editing/full-site-editing-plugin/ ~/Dev/wordpress/wp-content/plugins/full-site-editing-plugin
-
-ln -s ~/Dev/wp-calypso/apps/full-site-editing/blank-theme/ ~/Dev/wordpress/wp-content/themes/blank-theme
+/dist
```

Comment:
In addition to this one, it seems to be also creating another `dist` subfolder in `full-site-editing-plugin`. Is that intended? (asking because it was confusing for me at first)

<img width="318" alt="Screenshot 2019-04-23 at 17 47 25" src="https://user-images.githubusercontent.com/1182160/56596110-31964980-65f0-11e9-9b5c-d08074cffdb6.png">

Reply:
I believe that might be the `dist` folder created by the previous build system, which gets copied into the new `dist`.
Try deleting all three `dist` folders and rebuild: that shouldn't be there anymore. 🙂

Reply:
Yeah, I was suspecting I was doing something wrong and tried deleting them and rebuilding (seemed to be coming back). Anyway, probably an error on my end so feel free to proceed. :)